strapi/src/components/help/help-entry.json

```diff
@@ -0,0 +1,18 @@
+{
+  "collectionName": "components_help_help_entries",
```

Comment:
In other project, we migrated from feature-based groups (like "help") to two simple groups: "blocks" and "sections"